tests/tpostgres.nim

```diff
@@ -4,7 +4,7 @@ import times
 import unittest
 
 proc test_open(): DbConn =
-  open("", "postgres", "", "host=localhost port=5432")
+  open("postgres", "postgres", "", "postgres")
```

Comment:
I wish this could be still runnable outside of docker.
```suggestion
  open(getEnv("PGHOST", "postgres"), "postgres", "", "postgres")
```

Reply:
Applied the suggested change, updated the PR.